content/it/docs/concepts/configuration/_index.md

```diff
@@ -0,0 +1,4 @@
+---
+title: "Configurazione"
+weight: 80
+---
```

Comment:
the en version has a description, could you add it here as well?

Reply:
Added